scripts/exgfs_wave_prep.sh

```diff
@@ -192,10 +188,10 @@
          grdID=${WAVECUR_FID}
        ;;
        * )
-              echo 'Input type not yet implemented' 	    
-              err=3; export err;${errchk}
-              ;;
-     esac 
+         export err=3
+         err_chk 'Input type not yet implemented'
```

Comment:
@JessicaMeixner-NOAA Should this be a FATAL ERROR or WARNING?

Reply:
It should be a FATAL ERROR.  Will change.

Reply:
Fixed.